deprecate csi-doctor.sh and recommend kubectl jfs plugin (#1571)
juicedata/juicefs-csi-driver: docs/en/guide/dashboard.md

## Installation

:::warning Expose Web Service

> If your environment does not support exposing Web service through node port or Ingress, then CSI Dashboard is not an option. For this type of scenario, we recommend the [`kubectl jfs`](../administration/troubleshooting.md#kubectl-plugin) plugin, which is very powerful as well, you can use it to run many similar management and troubleshooting commands in terminal.

Reviewer: ```suggestion
If your environment does not support exposing a web service via node port or Ingress, CSI Dashboard is not an option. In such cases, we recommend the [`kubectl jfs`](../administration/troubleshooting.md#kubectl-plugin) plugin, which is also very powerful. You can use it to run many similar management and troubleshooting commands in the terminal.
```
